fe/fe-core/src/main/java/com/starrocks/load/streamload/StreamLoadMgr.java

```diff
@@ -101,12 +100,6 @@ public void init() {
         lock = new ReentrantReadWriteLock(true);
     }
 
-    public void beginLoadTask(String dbName, String tableName, String label, String user, String clientIp, long timeoutMillis,
```

Comment:
why remove this interface?

Reply:
it's duplicated when `warehouseid` param added by default, and after this pr, it's never used again

Reply:
Have added back the original interface.